vaadin-spring-tests/test-spring-common/src/main/java/com/vaadin/flow/spring/test/EmbeddedJettyConfiguration.java

```diff
@@ -0,0 +1,21 @@
+package com.vaadin.flow.spring.test;
+
+import javax.servlet.Servlet;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.util.Loader;
+import org.eclipse.jetty.webapp.WebAppContext;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnClass;
+import org.springframework.boot.web.embedded.jetty.JettyServerCustomizer;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConditionalOnClass({Servlet.class, Server.class, Loader.class,
```

Comment:
Used because there are some tests that uses tomcat instead of jetty. 

Based on https://github.com/spring-projects/spring-boot/blob/c52eabae22c0f553e6f3af2b5fe3753c2bbd79ad/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/web/servlet/ServletWebServerFactoryConfiguration.java

This class can be removed once the jetty "issue" is resolved.

Reply:
Which issue are you referring to ? At our end or in Spring Boot ?

It would be good to drop a comment to the code about the issue.

Reply:
See this comment https://github.com/spring-projects/spring-boot/issues/26634#issuecomment-859709434 and the linked issue